groth16/src/api.rs

```diff
@@ -148,6 +147,113 @@ pub fn groth16_verify(
 
     Ok(())
 }
+pub fn generate_verifier(vk_file_path: &str, sol_file_path: &str) -> Result<()> {
```

Comment:
Can we declare that we adopt those code from Zokrate?

Reply:
done